src/Npgsql/PublicAPI.Shipped.txt

```diff
@@ -8,7 +8,6 @@ abstract Npgsql.TypeHandling.NpgsqlSimpleTypeHandler<TDefault>.Write(TDefault va
 abstract Npgsql.TypeHandling.NpgsqlSimpleTypeHandlerWithPsv<TDefault, TPsv>.ReadPsv(Npgsql.NpgsqlReadBuffer! buf, int len, Npgsql.BackendMessages.FieldDescription? fieldDescription = null) -> TPsv
 abstract Npgsql.TypeHandling.NpgsqlSimpleTypeHandlerWithPsv<TDefault, TPsv>.ValidateAndGetLength(TPsv value, Npgsql.NpgsqlParameter? parameter) -> int
 abstract Npgsql.TypeHandling.NpgsqlSimpleTypeHandlerWithPsv<TDefault, TPsv>.Write(TPsv value, Npgsql.NpgsqlWriteBuffer! buf, Npgsql.NpgsqlParameter? parameter) -> void
-abstract Npgsql.TypeHandling.NpgsqlTypeHandler.CreateArrayHandler(Npgsql.PostgresTypes.PostgresArrayType! arrayBackendType) -> Npgsql.TypeHandlers.ArrayHandler!
```

Comment:
IIRC I don't think we're supposed to remove APIs from Shipped - but to prefix the removed API with *REMOVED* in the Unshipped file (this way Shipped always represent the current, shipped API surface, and Unshipped the latest/current unshipped state).

Let me know if this all seems annoying :) I'm interesting in how this analyzer helps (or doesn't) the the development process.

Reply:
You meant to write `*REMOVED*` 😁

Yeah, ATM it is indeed a bit annoying but that's mostly because it's new for me and documentation is a bit scarce, but I see great potential here, so I'm willing to learn.